intro_to_r/intro_to_r.Rmd

```diff
@@ -66,7 +82,7 @@ boys to newborn girls, so he gathered the baptism records for children born in
 London for every year from 1629 to 1710.  We can take a look at the data by 
 typing its name into the console.
 
-```{r view-data, eval=FALSE}
+```{r view-data}
 arbuthnot
```

Comment:
Yup :) And that's what all my packages are transitioning to. It's just so much more convenient if you have more than a couple of hundred rows of data.  (Also they shouldn't have to type `data(x)` because you should have `LazyLoad: true` in your `DESCRIPTION` so they can just type `x`)